tutorials/quickstart.eve.md

```diff
@@ -0,0 +1,92 @@
+# Quickstart
+
+In this guide we're going to give a 5 minute introduction to the essential concepts in Eve. If you've never used Eve before, you're in the right place.
+
+## Adding records to Eve
+
+Eve represents data as records, which are key value pairs attached to a unique ID. Eve programs are made up of small composable blocks that search for and create records. Let's start with a block that adds a record to Eve:
+
+```eve
+commit
+  [#greeting text: "hello world"]
+```
+
+This record is tagged `#greeting`, and has an attribute "text" with the value "hello world". Although tags have a special syntax, they are attributes like any other (i.e. #foo = tag: "foo"). We encourage you to classify groups of related records with tags.
+
+## Finding records in Eve
+
+We can search for records, then bind new records to those that are found. Eve finds every record matching the supplied patterns, and then binds records dependent on them. If no records match the search, then the block does not run at all. A block will only run if every pattern in a search matches at least one record. Let's search for the `#greeting` we just committed, and then display it in a text container:
+
+```eve
+search
+  [#greeting text]
+
+bind
+  [#ui/text text]
+```
+
+Variables with the same name are equivalent within a block; because they have the same name, the `text` in `[#greeting text]` and `[#ui/text text]` are equivalent. Go ahead and add another `#greeting` record to the first block to see what happens when more than one record matches the search (you can add it on a new line under the one that's there). 
+
+## Records update as data changes
+
+Blocks in Eve react automatically to changes in data. When a record changes, any bound records are automatically updated. Let's search for the current time, and display it in a div:
+
+```eve
+search
+  [#time seconds]
+
+bind 
+  [#ui/text text: seconds]
+```
+
+As the time changes, the output updates to reflect the current state of the `#time` record. Records can be committed instead of bound, but the behavior is a little different -- committed records persist until they are removed explicitly. Try changing `bind` to `commit` in the above block and see what happens.
+
+## Reacting to events
+
+Let's draw a button on the screen:
+
+```eve
+commit
+  [#ui/button #increment text: "+1"]
+```
+
+Every time you click this button, a record is created representing the click. You can react to this event by searching for the `#event/click` record:
+
+```eve
+search
+  event = [#event/click element: [#increment]]
+
+commit
+  [#clicked event]
+```
+
+Event records in Eve are removed immediately after they are created; however, because the `#clicked` record is committed, it will persist until explicitly removed. In Eve, `=` signifies equality (not assignment), so the variable `event` is a reference to the `#event/click` record. 
```

Comment:
"Clicks only last for an instant, but we want to create a permanent record of each click so we can search them later. For each click, a `#clicked` record is committed that will persist until it's explicitly removed. Much like the `#greeting` `text` we bound to the `#ui`, variables with the same name are equivalent, so the variable `event` in the `#clicked` record is a reference to the `#event/click` on the `#increment` button."